lib/scaffold.dart

```diff
@@ -499,6 +502,7 @@ class BackdropScaffoldState extends State<BackdropScaffold>
 
   Widget _buildFrontPanel(BuildContext context) {
     return Material(
+      color: this.widget.frontLayerBackgroundColor?? Colors.white,
```

Comment:
@farmery please remove the `?? Colors.white`.

Docs of `Material.color` say: "By default, the color is derived from the [type] of material."
So, if `widget.frontLayerBackgroundColor` is not set (= is null), `Material` handles the initialisation of the color itself.